### ISSUE TITLE
[WARN] Distutils has been deprecated

### ISSUE THREAD

**Issue by sevdog** [carbonblack/cbapi-python#319]
**I am seeing this behaviour on: (please complete the following information):**
 - Python Version: 3.9+

**Describe the bug**
To check server version cbapi relies on `distutils.version` (see [usage](https://github.com/carbonblack/cbapi-python/search?q=distutils)). This module has been [deprecated](https://peps.python.org/pep-0632/) and now it returns the following warning:

```
DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
```

**Steps to Reproduce**
Steps to reproduce the behavior (Provide a log message if relevant):
1. Istantiate any CbAPI class (`CbResponseAPI` or `CbProtectionAPI`)

**Expected behavior**
No warning should be raised.

**Comment by emitreva-cb** [comment]
We have released a new version of CbAPI (1.7.10), which is using packaging library for python3.7+. Could you please test with the latest version and provide feedback on whether the issue was fixed?

**Comment by sevdog** [comment]
@emitreva-cb I have tried the new release and now it does not raise any warning (in my environment, Python 3.9).

Looking at #320 it seems that it solves this issue. Thank you.
